src/main/java/it/pagopa/transactions/utils/NodoOperations.java

```diff
@@ -120,6 +104,9 @@ private Mono<String> nodoActivationForNM3PaymentRequest(
         request.setAmount(amount);
         request.setQrCode(qrCode);
         request.setIdempotencyKey(idempotencyKey);
+
+        LocalDate dueDate = LocalDate.now().plusYears(100);
```

Comment:
Open point: right now `dueDate` is modeled incorrectly inside `PaymentRequestInfo` (there is a wrong `@Pattern` annotation and most importantly a `@Nonnull` one). Here i populate it with a dummy value but `null` should be the correct choice. 

Should we:
 - have a PR remove the annotations on `dueDate` and then use `null` here
 - use `null` here directly (personally I don't like it very much as it would create an inconsistency between the model and how it's actuallty used)
 - keep the dummy value
?

Reply:
I would say that the first option is the better one.

Reply:
Option 1: pagopa/pagopa-ecommerce-commons#18